src/main/java/core/mate/academy/model/Bulldozer.java

```diff
@@ -5,6 +5,14 @@
  * Do not remove no-args constructor
  */
 public class Bulldozer extends Machine {
+    private int bulldoserPower;
+
+    public Bulldozer(String name, String color, int power) {
+        super.setName(name);
+        super.setColor(color);
```

Comment:
```suggestion
        super(name, color);
```
better create constructor (if it is not present add)